providers/aws/components/db/setup.ftl

```diff
@@ -191,10 +187,7 @@
         [/#if]
     [/#list]
 
-    [#local processorProfile = (getProcessor(occurrence, "RDS")?has_content)?then(
-                                getProcessor(occurrence, "RDS"),
-                                getProcessor(occurrence, DB_COMPONENT_TYPE )
-                            )]
+    [#local processorProfile = getProcessor(occurrence, "db" )]
```

Comment:
So distinguishing between types and profile names? No actual dependency - yes?

Reply:
Since each type is now defined in the reference configuration we can use the alternate name setup to handle this instead